windows/webview_bridge.cc

```diff
@@ -478,6 +481,36 @@ void WebviewBridge::HandleMethodCall(
     return result->Error(kErrorInvalidArgs);
   }
 
+  if (method_name.compare(kMethodAddScriptToExecuteOnDocumentCreated) == 0) {
+    if (const auto script = std::get_if<std::string>(method_call.arguments())) {
+      std::shared_ptr<flutter::MethodResult<flutter::EncodableValue>>
+          shared_result = std::move(result);
+
+      webview_->AddScriptToExecuteOnDocumentCreated(*script, [shared_result](bool success, LPCWSTR script_id) {
+        if (success) {
+          std::string sid = CW2A(script_id, CP_UTF8);

```

Comment:
Can you move the `LPCWSTR` -> `std::string` conversion to `webview.cc` and change the signature of `AddScriptToExecuteOnDocumentCreatedCallback` to accept `std::string`?